src/demo/demoUtils.js

```diff
@@ -183,9 +182,10 @@ export const commonSteps = {
 export const commonLanguages = {
   en: 'en',
   es: 'es',
-  'custom (fr)': {
-    locale: 'fr',
-    phrases: { 'welcome.title': 'Ouvrez votre nouveau compte bancaire' }
+  de: 'de',
+  fr: 'fr',
+  'custom': {
+    phrases: { 'welcome.title': 'My custom title' }
```

Comment:
Does this mean that `locale` within the 'custom' object was never really needed? Just wondering.

Reply:
If `locale` is not passed, it will default to `en` but some clients might not want to use the default locale. For example, when `fr` was not supported it was actually possible to use `fr` as custom translation to be able to send a `fr` to telephony.